lib/CLI.js

```diff
@@ -2369,7 +2369,7 @@ function checkDeprecates(conf){
  */
 function prepareAppName(conf){
   if (!conf.name){
-    conf.name = p.basename(conf.script);
+    conf.name = conf.script !== undefined ? p.basename(conf.script) : 'undefined';
```

Comment:
Yes, I agree. I'm not familiar with your code base though so I'm open to any suggestion on how to fix it properly.